config/src/main/java/org/hyperledger/besu/config/JsonBftConfigOptions.java

```diff
@@ -34,6 +34,8 @@ public class JsonBftConfigOptions implements BftConfigOptions {
 
   private static final long DEFAULT_EPOCH_LENGTH = 30_000;
   private static final int DEFAULT_BLOCK_PERIOD_SECONDS = 1;
+  private static final int DEFAULT_EMPTY_BLOCK_PERIOD_SECONDS = 0;
+  // 0 keeps working as before, increase to activate it
```

Comment:
I feel like this comment should be on the line before the constant, not after it?

Reply:
I had it after the the constant, but then I saw other existing comments after the lines they refer to so I thougth it was some kind of style... I'll move to the line above.

Reply:
Actually, that line I saw is the one below, right after the DEFAULT_ROUND_EXPIRY_SECONDS ...